### ISSUE TITLE
[enhancement] Remove unnecessary windows when creating new objects

### ISSUE THREAD

**Issue by Ruslan-Aleev** [modxcms/revolution#14373]
### Summary
As @sottwell correctly pointed out in media sources (https://github.com/modxcms/revolution/issues/12853), there is an unobvious step in creating the source, **but the problem is more widespread**.

In MODX, when creating object, a quick creation form first pops up (usually with a name, description), for example, for contexts, see the picture below.

![create_1](https://user-images.githubusercontent.com/12523676/52708368-5d1f0700-2fa3-11e9-8fdc-23beca1169ff.png)

Then, to fully configure the context, you have to edit it in an expanded form through the context menu, see the picture below.

![create_2](https://user-images.githubusercontent.com/12523676/52708378-627c5180-2fa3-11e9-8e4a-334419ed51eb.png)

**Question: Why do I need a quick creation form, if all its fields (name, description, etc.) I can fill in an extended form?**

Quick create form creates unnecessary steps and clicks. And for users it is not clear what to do next, because an object appeared in the list, but how to configure it? There are no icons, there are no links, we already discussed this earlier - https://github.com/modxcms/revolution/issues/14125

This behavior with a quick creation form is in media sources, contexts, access settings, and most likely in other sections.
The logical behavior of the button (without a quick form) can be seen in the "Users" section - /manager/?a=security/user

**Comment by ant-tar** [comment]
Good catch, as far as I remember it was discussed twice some time ago on forums.modx.com too.
I am absolutely convinced that the quick window here is redundant and can be removed. When you work with a lot of contexts inside one website this becomes noticeable and waste time.

**Comment by petyagrill** [comment]
I think that the most optimal way is to redirect to the edit page after creating a media source or context in this window.

**Comment by Ruslan-Aleev** [comment]
@petyagrill In my opinion this is not a very good option, because it will not be clear where the extended form came from, and the behavior is not normal and it will take longer.

**Comment by petyagrill** [comment]
@Ruslan-Aleev  The method I proposed is simpler to implement, because in the case of a redirect, we will have an id of the entity and the rest of the functionality is already working. But if you click on the button to open an extended form (which you first need to create), then you will still need to redirect the first time you save it. What you offer is very similar to the creation of resources, but I believe that to solve this problem there is no need for such large changes.

**Comment by Ruslan-Aleev** [comment]
@petyagrill Unfortunately, I do not know the technical limitations for opening an extended form, perhaps you are right and this is not easy to implement.
The logical behavior of the button (without a quick form) can be seen in the "Users" section - /manager/?a=security/user
